### PR TITLE
feat & refactor: misc blog leftover tasks

### PR DESCRIPTION
**Pull request by k8isdead**
**Describe changes**

This PR brings various tasks [from this Notion page](https://www.notion.so/pixelpoint/Blog-leftovers-bb1c6b6dc19e439f9c62243188c8eb5e#3504b1e5232b4644bf08b2218e117723)

**Steps to test**
1. Go to [_/privacy-policy_](https://clever-wescoff-fd0120.netlify.app/privacy-policy) page as an example of a static page in this project
2. Confirm everything works and looks as expected

NB! Please review the code locally, as deploy preview is unavailable for now. 

**TODO**

- [x] Find a way to make embedded twitter card make responsive 
- [ ] Delete all posts before merging PR and leave only one as an example without much content
- [ ] Respond to additional designers and devs comments, if there are any

### CODE REVIEW

**Comment by kelvich** [comment]
@k8isdead @SilencerWeb @lnikell 

Hey! Sorry for breaking deploys for some time. Cloudflare supports only one build command for all of the branches, and with docs it looks like that:
```
gatsby build --prefix-paths; sh ./build_docs.sh
```
so branches without `build_docs.sh` were broken for some time.

Now I've committed that script both to `main` and `new` branches. So deploy should work after the rebase:

```
git fetch -a
git rebase origin/new
git push -f
```

Small overview of docs build is available in readme https://github.com/zenithdb/website/tree/new#documentation I think now you can ignore it totally for gatsby development.

File: gatsby-config.js [hunk 1]
```diff
@@ -74,6 +74,13 @@ module.exports = {
         path: `${__dirname}/src/posts`,
       },
     },
+    {
+      resolve: 'gatsby-source-filesystem',
+      options: {
+        name: 'posts',
```
Comment (SilencerWeb) [inline gatsby-config.js:80]:
```suggestion
        name: 'static-pages',
```

File: gatsby-node.js [hunk 2]
```diff
@@ -10,6 +10,31 @@ const getBlogPostPath = require('./src/utils/get-blog-post-path');
...
+
+  staticPages.forEach(({ id, slug }) => {
+    actions.createPage({
+      path: `/${slug}`,
+      component: path.resolve('./src/templates/static.jsx'),
+      context: { id },
+    });
+  });
+}
+
```
Comment (SilencerWeb) [inline gatsby-node.js:37]:
```suggestion
async function createStaticPages({ graphql, actions }) {
  const result = await graphql(`
    query staticPageQuery {
      allMdx(filter: { fileAbsolutePath: { regex: "/src/static-pages/" } }) {
        nodes {
          id
          slug
        }
      }
    }
  `);

  if (result.errors) throw new Error(result.errors);

  result.data.allMdx.nodes.forEach(({ id, slug }) => {
    actions.createPage({
      path: `/${slug}`,
      component: path.resolve('./src/templates/static.jsx'),
      context: { id },
    });
  });
}

```

Comment (SilencerWeb) [reply gatsby-node.js:37]: Also, let's move this function right after `createBlogPosts`

File: src/static-pages/privacy-policy.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
+---
+title: 'Zenith — Privacy Policy'
+---
+
+## Privacy Policy
```
Comment (SilencerWeb) [inline src/static-pages/privacy-policy.md:5]:
```suggestion
# Privacy Policy
```

File: src/templates/static.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import SEO_DATA from 'constants/seo-data';
+
+const StaticPage = ({
+  data: {
+    mdx: { body, frontmatter },
+  },
+}) => (
+  <Layout seo={SEO_DATA.static({ title: frontmatter.title })} headerTheme="white">
+    <article>
+      <div className="safe-paddings py-48 3xl:py-44 2xl:py-40 xl:py-32 lg:py-12 md:py-6">
```
Comment (SilencerWeb) [inline src/templates/static.jsx:16]: Why do we have this unnecessary wrapper?

File: src/templates/static.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/prop-types */
+import { graphql } from 'gatsby';
+import React from 'react';
+
+import Content from 'components/shared/content';
+import Layout from 'components/shared/layout';
+import SEO_DATA from 'constants/seo-data';
+
+const StaticPage = ({
```
Comment (SilencerWeb) [inline src/templates/static.jsx:9]:
```suggestion
const StaticTemplate = ({
```

File: src/static-pages/privacy-policy.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
+---
+title: 'Zenith — Privacy Policy'
```
Comment (SilencerWeb) [inline src/static-pages/privacy-policy.md:2]: image

File: src/templates/static.jsx (hunk 4 above)
Comment (SilencerWeb) [reply src/templates/static.jsx:16]: Also, please think about why bottom padding is wrong and what values we should use

**Review by SilencerWeb — CHANGES_REQUESTED** [review]

**Review by SilencerWeb — APPROVED** [review]

**Pull request merged**